src/index.ts

```diff
@@ -17,17 +17,19 @@ const renderUsers = async (config: ApiConfig) => {
   });
 };
 
-const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
-
-  renderUsers(config.api);
+const app = (ioc: any) => {
+  const usersService = ioc.resolve('Users');
+  renderUsers(usersService);
 };
 
 window.onload = (event: Event) => {
-  const logger = new Logger();
+  const config = (window as any).__CONFIG__;
+  delete (window as any).__CONFIG__;
+
+  const ioc = createIoCContainer(config.api);
```

Comment:
Can you create an ioc container in the root of the file ? So you can use a container without passing it as parameter and even import it somewhere.